### PR TITLE
[DEV-1141, DEV-1142] Signup and Confirm signup copy

### PR DESCRIPTION
**Pull request by jeremygordillo**
#### List of Changes

Updated signup page and confirm signup page copy

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by marcobottaro** [comment]
I see some differences in page layout. Can you please fix the title's colors, turn off the `text-transform: uppercase` property, and check the form box padding?
screenshot-www figma com-2023 11 23-11_01_35

**Review by marcobottaro — CHANGES_REQUESTED** [review]
https://github.com/pagopa/developer-portal/pull/442#issuecomment-1824114217

**Comment by marcobottaro** [comment]
Can you please also check the style of this part (it partially differs from login's and from the design in Figma)
image
image
image

**Review by marcobottaro — APPROVED** [review]

**Comment by kin0992** [comment]
@jeremygordillo please do follow the copy within the Jira activity: some of the points must be coming soon and right now we can add the "in arrivo" string at the end of the title

File: apps/nextjs-website/src/_contents/translations.ts [hunk 1]
```diff
@@ -359,16 +359,20 @@ export const translations = {
...
+          text: "Non perdere neanche un nuovo contenuto, guida o tutorial. Puoi scegliere di ricevere via email le novità sui prodotti PagoPA e sull'integrazione tecnologica. ",
         },
         {
-          title: 'vantaggio 3',
-          text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+          title: 'Accedi ai tool per velocizzare i tuoi sviluppi',
+          text: 'Sblocca tutto il potenziale del DevPortal con API Key, Mocker, SDK ed altro ancora.',
+        },
+        {
+          title: 'Richiedi assistenza su un canale dedicato',
```
Comment (kin0992) [inline apps/nextjs-website/src/_contents/translations.ts:374]:
```suggestion
          title: 'Richiedi assistenza su un canale dedicato (in arrivo)',
```

**Review by kin0992 — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]

**Pull request merged**